pages/agent/v3/aws/autoscaling_mac_metal.md.erb

```diff
@@ -0,0 +1,200 @@
+# Auto Scaling mac1.metal instances
+
+Run your builds on AWS `mac1.metal` using our [CloudFormation template](https://github.com/buildkite/elastic-mac-for-aws).
+This template creates an Auto Scaling group, Launch Template, and Host Resource Group
+suitable for maintaining a pool of EC2 `mac1.metal` instances that run the
+Buildkite Agent. These agents can be used to run Buildkite Pipelines and build
+Xcode based software projects for macOS, iOS, iPadOS, tvOS, and watchOS.
+
+As you must prepare and supply your own AMI for this template, macOS support has
+**not** been incorporated into the Elastic CI Stack for AWS.
+
+Using an Auto Scaling group for your instances ensures booting your macOS
+Buildkite Agents is repeatable, and provides automatic instance replacement when
+hardware failures occur.
+
+## Prerequisites
+
+You should have familiarity with:
+
+* AWS VPCs
+* AWS EC2 AMIs
+* macOS GUI
+
+You must also choose an AWS Region with `mac1.metal` instances available. See
+[Amazon EC2 Mac Instances](https://aws.amazon.com/ec2/instance-types/mac/) and
+[Amazon EC2 Dedicated Hosts Pricing](https://aws.amazon.com/ec2/dedicated-hosts/pricing/)
+for details on which regions have `mac1.metal` dedicated hosts.
+
+See also the [Amazon EC2 Mac instances User Guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-mac-instances.html)
+for more details on AWS EC2 Mac instances.
+
+## Choose a VPC Layout
+
+Before deploying this template you must choose a VPC subnet design, and which
+VPC security groups your instances should belong to.
+
+Depending on your threat model, you may find running instances in your default
+VPC’s public subnets with a public IP address suitable. Otherwise, you may wish
+to explore options like separate Public/Private subnets and a NAT Gateway, and
+using a Bastion or VPN to access the instances over SSH and VNC.
+
+In supported regions, `mac1.metal` dedicated hosts may not be available in
+every Availability Zone. Consider using a subnet in every Availability Zone to
+maximise the pool of instances available to boot from.
+
+You also need to configure or define the VPC Security Groups your instance
+network interfaces should belong to. At a minimum, inbound SSH access is
+required to set up your initial template AMI.
+
+## Build an AMI
+
+Before deploying this template, you must create a template AMI that can be
+horizontally scaled across multiple instances.
+
+1. Reserve a [AWS mac1.metal](https://aws.amazon.com/ec2/instance-types/mac/)
+Dedicated Host.
+1. Boot a macOS instance using your desired AMI on the Dedicated Host.
+1. Configure instance VPC subnets, security groups, and key pairs so that you
+can access the instance.
+1. Using an SSH or SSM session:
+	* Set a password for the ec2-user using `sudo passwd ec2-user`
```

Comment:
These steps are not required to set up the template AMI. I recommend using private key authentication. Personally I would not enable password authentication for sshd as it is vulnerable to brute force attacks, something far harder with private key authentication, but it may be suitable for your use case depending on your threat model and hosts’ network position.